adhocracy4/filters/widgets.py

```diff
@@ -61,6 +61,12 @@ def render(self, name, value, attrs=None, choices=(), renderer=None):
         })
 
 
+class OrderingWidget(DropdownLinkWidget):
+    """Used in .filters.OrderingFilter"""
```

Comment:
This should be DynamicChoicesOrderingFilter, right?

Reply:
Right, good catch! I renamed the filter while moving and added the new comment at the same time and then did not update that...